README.md

```diff
@@ -10,46 +10,120 @@ It's important to note that the code in this repo may differ from the code writt
 ## Lab 00
 Running Fusion locally.
 
-1. Clone this repo
+1. Checkout your own branch
```

Comment:
This is misleading as it could mean checkout from lab-00. I think they dont have to checkout their own branch at all. Once they start on master, they can stay on master.

Reply:
That's true, in which case should we eliminate the idea of the username-working branch entirely?